scripts/changelog_check.rb

```diff
@@ -134,7 +135,7 @@ def generate_changelog(git_log)
 
     changelog_entry = ChangelogEntry.new(
       category: category,
-      subcategory: change[:subcategory].capitalize,
+      subcategory: change[:subcategory].upcase_first,
```

Comment:
I didn't realize this inflection change was here at all... maybe we just remove it?

Reply:
> I didn't realize this inflection change was here at all... maybe we just remove it?

That was my first approach, and I'm still open to it. It was the test case that convinced me, since I could definitely see someone writing a changelog like this:

https://github.com/18F/identity-idp/blob/75c5dbb40863ccb4ed24be6eabcd2c181f766939/spec/scripts/changelog_check_spec.rb#L98

Then again, I ran the script on the real changelog after removing `capitalize` and it looked good, so maybe it's prematurely optimizing for a problem that doesn't really exist. 🤷 

I think it's also an interesting question if / how much to auto-format. I had planned to discuss some ideas in the engineering huddle later today (e.g. fixing common typos, normalizing subcategory names, removing ticket references, etc).